### PR TITLE
Add/Fix conversionMultipliers

### PR DESCRIPTION
**Pull request by fkleedorfer**
Checked for errors in conversionMultipliers, or missing ones that can be inferred.

### CODE REVIEW

**Comment by fkleedorfer** [comment]
We can talk about the numeric representation. I can recreate the added triples pretty easily, so if you want a different precision or if you prefer scientific notation, just say so.

(Side glance at #821regarding scientific notation)

**Comment by fkleedorfer** [comment]
Rebased onto latest revision of main

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -31338,3 +31278,1243 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
...
+  #       └──┬ unit:J multiplier: 1.0
+  #          ├─── unit:M multiplier: 1.0
+  #          └──┬ unit:N multiplier: 1.0
+  #             ├──┬ unit:KiloGM multiplier: 1.0
+  #             │  └─── unit:GM multiplier: 0.001
+  #             ├─── unit:M multiplier: 1.0
+  #             └─── unit:SEC^-2 multiplier: 1.0 (multiplier^-2: 1.0)
+unit:PER-WB qudt:conversionMultiplier 1.0 .
+
+  # unit:MilliR has conversionMultiplier 2.58E-7, but we've found out that 1 mR = 3.33564095198152049575576714474920E-13 C/kg (relative diff: 1.999994828470326527163271547239499 - big difference)
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:32338]: This one seems wrong to me. Could you confirm?

Comment (steveraysteveray) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:32338]: ...and if it is wrong, all the units involving the roentgen need to be adjusted.

**Review by steveraysteveray — COMMENTED** [review]
Impressive piece of work. I spot-checked some units and it looks good with the exception of the roentgen units. Appreciate you double-checking those - see embedded comment.

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl (hunk 1 above)
Comment (fkleedorfer) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:32338]: I think you are right, good catch!

I've rebased onto main and removed the changes for R, MilliR and KiloR

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl [hunk 2]
```diff
@@ -31434,3 +31376,1211 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "QUDT Unit of Measure Vocabulary Metadata Version 2.1.34" ;
 .
+
+
+
+  # random correction
+unit:MilliR_man
+    qudt:isScalingOf unit:SV ;
+    qudt:conversionMultiplier 0.0001 .
```
Comment (steveraysteveray) [inline vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:31385]: System choked on the trailing .
Should be ;

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
See the one trivial syntax error.

File: vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl (hunk 2 above)
Comment (fkleedorfer) [reply vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:31385]: Fixed. And I removed the qudt:isScalingOf triple, as these have been removed from QUDT long ago. Qudtlib still relies on them for building the factor unit hierarchy, so I've got a separate file with these - this triple slipped into the PR by accident.

**Comment by fkleedorfer** [comment]
done

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**